client/components/seo-preview-pane/index.jsx

```diff
@@ -62,6 +64,29 @@ const getPostImage = ( post ) => {
 		: null;
 };
 
+const getSeoExcerptForPost = ( post ) => {
+	if ( ! post ) {
+		return null;
+	}
+
+	return formatExcerpt( find( [
+		PostMetadata.metaDescription( post ),
+		post.excerpt,
+		post.content
+	], identity ) );
+};
+
+const getSeoExcerptForSite = ( site ) => {
+	if ( ! site ) {
+		return null;
+	}
+
+	return formatExcerpt( find( [
+		site.options.seo_meta_description,
```

Comment:
I went with using `get`, I couldn't quite grok your suggestions which is my fault entirely ;)